src/gui/menu.hpp

```diff
@@ -78,7 +79,7 @@ class Menu
 
   ItemHorizontalLine& add_hl();
   ItemLabel& add_label(const std::string& text);
-  ItemAction& add_entry(int id, const std::string& text);
+  ItemAction& add_entry(int id, const std::string& text, FontPtr font = Resources::normal_font);
```

Comment:
There shouldn't be a need to provide the font in the `Menu` constructor function. Could be provided on the returned `ItemAction`.